fabric-gamerules-v1/src/main/java/net/fabricmc/fabric/api/gamerule/GameRuleUtils.java

```diff
@@ -0,0 +1,50 @@
+package net.fabricmc.fabric.api.gamerule;
+
+import com.mojang.brigadier.arguments.ArgumentType;
+import net.fabricmc.fabric.impl.gamerule.GameRuleUtilsImpl;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.util.Identifier;
+import net.minecraft.world.GameRules;
+
+import java.util.function.BiConsumer;
+import java.util.function.Function;
+import java.util.function.Supplier;
+
+public interface GameRuleUtils {
```

Comment:
This looks a lot more like a registry than a generic utility class, doesn't it ? Utils classes generally have static methods only.

Reply:
Yeah, that's fair. I was more feeling a bit iffy on calling it a registry because 3/4 of its methods are utility methods, but it should be talked over a bit more

Reply:
Maybe name it `FabricGameRules` or `GameRules`

Reply:
I don't see why this is an interface at all? This API could just be all static, there is no instance dependency

Reply:
It's the same for every other class in the API.